token/cli/Cargo.toml

```diff
@@ -26,12 +27,20 @@ solana-logger = "=1.16.13"
 solana-remote-wallet = "=1.16.13"
 solana-sdk = "=1.16.13"
 solana-transaction-status = "=1.16.13"
-spl-token = { version = "4.0", path="../program", features = [ "no-entrypoint" ] }
-spl-token-2022 = { version = "0.8", path="../program-2022", features = [ "no-entrypoint" ] }
-spl-token-client = { version = "0.6", path="../client" }
-spl-token-metadata-interface = { version = "0.2", path="../../token-metadata/interface" }
-spl-associated-token-account = { version = "2.0", path="../../associated-token-account/program", features = [ "no-entrypoint" ] }
-spl-memo = { version = "4.0.0", path="../../memo/program", features = ["no-entrypoint"] }
+spl-token = { version = "4.0", path = "../program", features = [
+  "no-entrypoint",
+] }
+spl-token-2022 = { version = "0.8", path = "../program-2022", features = [
+  "no-entrypoint",
+] }
+spl-token-client = { version = "0.6", path = "../client" }
+spl-token-metadata-interface = { version = "0.2", path = "../../token-metadata/interface" }
+spl-associated-token-account = { version = "2.0", path = "../../associated-token-account/program", features = [
+  "no-entrypoint",
+] }
+spl-memo = { version = "4.0.0", path = "../../memo/program", features = [
+  "no-entrypoint",
+] }
```

Comment:
Did your formatter pick these up? I haven't seen this before!

Reply:
Yeah I recently updated my neovim config (I now use [lazyvim](https://www.lazyvim.org/)) to enable lsp-server with [taplo](https://taplo.tamasfe.dev/) and it automatically validates cargo files as in an IDE, which is pretty cool. I also enabled auto-[format](https://taplo.tamasfe.dev/cli/usage/formatting.html) on save as well. I guess should have filtered these format changes since it is orthogonal to this PR, sorry 😅.